apiv6/generate_client.sh

```diff
@@ -1,52 +1,79 @@
 #!/usr/bin/env bash
+# Generate Go client SDK from OpenAPI specification.
 
-# Generate golang client sdk from OpenAPI 3.0.0 spec
+openapi_generator_tag="v5.1.1"
 
+cloud_export_spec_filename="cloud_export.openapi.json"
+cloud_export_spec_url="https://raw.githubusercontent.com/kentik/api-schema-public/master/gen/openapiv2/kentik/cloud_export/v202101beta1/cloud_export.swagger.json"
+synthetics_spec_filename="synthetics.openapi.json"
+synthetics_spec_url="https://raw.githubusercontent.com/kentik/api-schema-public/master/gen/openapiv2/kentik/synthetics/v202101beta1/synthetics.swagger.json"
 
-function stage() {
-    BOLD_BLUE="\e[1m\e[34m"
-    RESET="\e[0m"
-    msg="$1"
-    
-    echo
-    echo -e "$BOLD_BLUE$msg$RESET"
+function run() {
+    check_prerequisites
+
+    download_openapi_spec
+    generate_cloud_export_client
+    generate_synthetics_client
+    go fmt ./...
 }
 
-function checkPrerequsites() {
+function check_prerequisites() {
     stage "Checking prerequisites"
-    
+
     if ! docker --version > /dev/null 2>&1; then
-        echo "You need to install docker to run the generator"
+        echo "You need to install Docker to run the generator"
         exit 1
     fi
 
     echo "Done"
 }
 
-function generate_golang_client_from_openapi3_spec() {
-    stage "Generating golang client from openapi spec"
+function download_openapi_spec() {
+    wget "$cloud_export_spec_url" --output-document "$cloud_export_spec_filename"
```

Comment:
Won't work if user does not have `wget` (I for one don't have it). I suggest to use `curl` which is more common and more versatile.

Reply:
Switched to `curl` and added it to prerequisites